docs/_static/usecase/v1.yaml

```diff
@@ -0,0 +1,190 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Use Case List Service API spec
+  description: API specification for use case list services available within nuts node
+  version: 1.0.0
+  license:
+    name: GPLv3
+servers:
+  - url: http://localhost:1323
+paths:
+  /usecase/{listName}:
+    parameters:
+      - name: listName
+        in: path
+        required: true
+        schema:
+          type: string
+    get:
+      summary: Retrieves a use case list
+      description: |
+        An API provided by the use case maintainer to retrieve a use case list, starting at the given lamport timestamp.
+        The client should provide the timestamp it was returned in the last response.
+        If no timestamp is given or it is 0, it will return the full list.
+
+        error returns:
+        * 400 - incorrect input
+      operationId: getList
+      tags:
+        - usecase
+      parameters:
+        - name: timestamp
+          in: query
+          schema:
+            type: integer
+            minimum: 0
+      responses:
+        "200":
+          description: Requested list is returned, alongside the timestamp which should be provided at the next query.
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - timestamp
+                  - entries
+                  - tombstones
+                properties:
+                  timestamp:
+                    type: integer
+                    minimum: 0
+                  entries:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/VerifiablePresentation"
+                  tombstones:
+                    description: |
+                      List of presentation IDs that were removed from the list.
```

Comment:
since the last query and/or non-expired VPs that have been removed?

Reply:
Replaced with empty presentation (of special type)